docs/consensus/pot.md

```diff
@@ -0,0 +1,58 @@
+Subspace's consensus obtains a farming dynamic that mimics the random time interval of Bitcoin's mining dynamic while only expending a small constant amount of electricity. This is achieved through a Proof-of-Time for block proposal lottery based on the fundamental paper [PoSAT: Proof-of-Work Availability and Unpredictability, without the Work](https://arxiv.org/abs/2010.08154) by Soubhik Deb, Sreeram Kannan and David Tse. It ensures the fairness of the farming process to all participants through complete unpredictability of who will get to propose a block next. This unpredictability is at the same level as PoW protocols and is stronger than existing PoStake protocols using Verifiable Random Functions.
+
+## Long-range attack
+
+Unlike in Proof-of-Work, the process of block production in Proof-of-Stake and Proof-of-Capacity-based
+blockchains is not physically constrained. This opens the door to a _long-range attack_.
+
+A long-range attack refers to an adversary controlling enough resources at some point of chain life to rewrite a significant portion of the chain history. In PoW protocols like Bitcoin, this requires controlling over 50% of the total network hashrate for a sustained period of time which is infeasible in practice. However, long-range attacks remain a serious threat in alternative consensus protocols that do not rely on proof-of-work.
+Let's show how a Proof-of-Space blockchain can be vulnerable to long-range attacks. An equivalent setup is also true for Proof-of-Stake protocols.
+Suppose in the first year of the blockchain's operation, all farmers were honest and have collectively pledged 100TB of storage. Now, suppose by the second year the total storage pledged reached 1PB, out of which an adversary has dedicated 200TB worth of storage. At no point does an adversary control more than 20% of storage, which is way less than a majority. However, using his 200TB, an adversary could rewrite the past year's history by participating in all past lotteries to win blocks back to the genesis and then grow a chain instantaneously from the genesis to surpass the current longest chain. This is possible because the adversary's resources are enough to win a disproportionately large number of past lotteries compared to his share of total storage. Such a long-range rewrite seriously threatens the security and immutability of blockchain history.
+On the other hand, in Bitcoin it takes a long time to mine such a chain from the past and that chain will always be behind the current longest chain. Thus, PoW enforces an arrow of time, meaning nodes cannot “go back in time” to mine blocks. This property is key to tolerating a fully dynamic honest and adversarial participation.
+
+## Arrow of Time
```

Comment:
I'm not sure this title is proper. Here we explain how Subspace prevents the above.

Reply:
How about Solution? Or Subspace Arrow-of-Time? Or just merge with prev section

Reply:
Subspace solution or approach?